apps/finance/app/package.json

```diff
@@ -46,11 +46,15 @@
   },
   "scripts": {
     "lint": "eslint ./src",
-    "start": "npm run sync-assets && npm run watch:script & parcel serve index.html -p 3002 --out-dir build/",
-    "build": "npm run sync-assets && npm run build:script && parcel build index.html --no-cache --out-dir build/ --public-url \".\"",
-    "build:script": "parcel build src/script.js --out-dir build/",
-    "watch:script": "parcel watch src/script.js --out-dir build/ --no-hmr",
-    "sync-assets": "copy-aragon-ui-assets -n aragon-ui ./build && rsync -rtu ./public/ ./build",
+    "test": "jest",
+    "start": "npm run sync-assets && npm run watch:script & parcel serve index.html -p 3002 --out-dir ../dist/",
+    "watch": "npm run watch:script",
+    "build": "npm run sync-assets && npm run build:app && npm run build:script",
+    "build:app": "parcel build index.html -d ../dist/ --public-url \".\" --no-cache",
+    "build:script": "parcel build src/script.js --out-dir ../dist/ --no-cache",
+    "serve": "parcel serve index.html --out-dir ../dist/ --no-cache",
```

Comment:
Yes as I said above we need it. And I agree, it makes to use 3002 👍

Reply:
Doing further tests if we want to change to port we need to do it on `buidler.config` file otherwise the app don't load correctly.

Reply:
Would like to understand how this works a bit better. Ideally `npm run start` would be similar to how the buidler plugin starts the app.

If I understand correctly, `serve`, `watch`, and `sync-assets` are requirements for buidler, but `start` is not. `start` is only used if we'd like to run the frontend "headless" by itself? And the port settings for serving the frontend with buidler **must** be configured in the buidler config?

If these are true, I'm starting to wonder if we should make the frontend's `start` script provide a help text of sorts, to better guide the user on which to run in case they ran it by mistake.

Reply:
Yes, all those you mentioned are true. You are right about the message, maybe also using another name instead of start?